libbeat/docs/shared-autodiscover.asciidoc

```diff
@@ -257,6 +257,39 @@ the ones used for discovery probes, they have these settings:
 include::../../{beatname_lc}/docs/autodiscover-jolokia-config.asciidoc[]
 endif::autodiscoverJolokia[]
 
+ifdef::autodiscoverAWSELB[]
+[float]
+===== Amazon ELBs
+
+The Amazon ELB autodiscover provider discovers ELBs and their listeners. This is useful when you don't want to connect
+directly to a service, but rather to the ELB fronting a pool of services.
+
+This provider will yield one config block per ELB Listener. So, if you have one ELB exposing both ports 80 and 443, it
+will generate two configs, one for each port. Keep in mind that the beat will de-duplicate configs. So, if the generated
+configs are the same only one will actually run.
+
+This provider will load AWS credentials using the standard AWS environment variables and shared credentials files see https://docs.aws.amazon.com/general/latest/gr/aws-access-keys-best-practices.html[Best Practices for Managing AWS Access Keys] for more information. If you do not wish to use these, you may explicitly set the `credentials.access_key_id` and `credentials.secret_access_key` variables.
+
+These are the available fields on every event
+
+  * host
```

Comment:
Not directly but I thought worth mentioning: This is not following ECS but is the same in other autodiscovery providers. I wonder if long term this could become an issue if we would also ingest some autodiscovery data for later querying.

Reply:
The `host` and `port` fields aren't inserted into the event. Only the `elb_listener` fields are. The fields listed here are what are present in the templating phase of the config. I'll add info here to clarify.

Reply:
I'm aware that it's not added to the event, but I'm still thinking if we should potentially align to prevent confusion? @exekias ?

Reply:
Sorry, I think there is some confusion here. Those fields *do* contain their own `host` and `port` fields. Additionally, if using heartbeat you'd probably use `url.port` and `url.domain` anyway.